_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -54,7 +54,7 @@ This rule applies to the first `meta` element in a document for which all the fo
 
 ## Expectation
 
-For each test target, running the [shared declarative refresh steps][], given the target's document, the value of the target's `content` attribute, and the target results in _time_ being either 0 or greater than 72000 (20 hours).
+For each test target, running the [shared declarative refresh steps][], refresh steps results in _time_ being 0 or greater than 72000 (20 hours).
```

Comment:
I'm not sure about removing the paramters from the expectation.
The [shared declarative refresh steps](https://html.spec.whatwg.org/multipage/semantics.html#shared-declarative-refresh-steps) state:
> The shared declarative refresh steps, given a [Document](https://html.spec.whatwg.org/multipage/dom.html#document) object _document_, string _input_, and optionally a [meta](https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element) element _meta_, are as follows

I'm not sure it is unambiguous enough to call on them without specifying _document_, _input_, and _meta_. Especially when we do then refer to _time_, which is declared in that algorithm (to store the result of the computation).

(typically, _document_ could be the target's document or it top-level browsing context document)